docs/questions/sharing/public-links.md

```diff
@@ -1,43 +1,47 @@
 ---
-title: Public links
+title: Public sharing
 redirect_from:
   - /docs/latest/administration-guide/12-public-links
   - /docs/latest/embedding/12-public-links
+  - /docs/latest/questions/sharing/public-links
 ---
 
-# Public links
+# Public sharing
 
 Sometimes you'll want to share a dashboard or question you've saved with someone that isn't a part of your organization or company, or someone who doesn't need access to your full Metabase instance. Metabase lets administrators create public links and [public embeds](#public-embed) to let you do just that.
 
-## Turning public links on
+## Enable public sharing on Metabase
 
 ![Enable public sharing](../images/enable-public-sharing.png)
-First things first, you'll need to go to the Admin Panel and enable public sharing. In the future, you'll see dashboards and questions you've shared listed here, and you'll be able to revoke any public links that you no longer want to be used.
+
+An admin needs to enable public sharing from **Settings** > **Admin settings** > **Public sharing** before you'll see the [sharing icon](#enable-sharing-on-your-saved-question-or-dashboard) option on questions and dashboards.
+
+Once toggled on, the **Public sharing** section will display Metabase dashboards and questions with active public links. To make a public link inactive, click **Revoke link**.
 
 ## Enable sharing on your saved question or dashboard
 
-![Enable sharing](../images/enable-links.png)
+If [public sharing](#enable-sharing-on-your-saved-question-or-dashboard) is enabled for your Metabase, you'll find the `Sharing and Embedding` icon on saved questions and dashboards (it looks like an arrow pointing up and to the right). The sharing button is located on the bottom right corner of a question, or the top right corner of a dashboard.
 
-Next, exit the Admin Panel and go to question that you want to share, then click on the `Sharing and Embedding` icon in the bottom-right of the screen (it looks like an arrow pointing up and to the right). Then click on the toggle to enable public sharing for this question.
+Click on the sharing icon to bring up the sharing settings modal, then click the toggle to generate the sharing link for that question or dashboard.
```

Comment:
```suggestion
To generate the sharing link for that question or dashboard, click on the sharing icon to bring up the sharing settings modal, then click the toggle.
```

We should state the action first, so people can decide to keep reading or bail if the action is irrelevant to their goal.